components/saved-apprenticeships.tsx

```
@@ -0,0 +1,112 @@
+        <h1 className="text-3xl font-bold text-gray-900 dark:text-white mb-8">
+          Saved Apprenticeships
+        </h1>
+
+        {savedListings.length === 0 ? (
+          <div className="bg-white dark:bg-gray-800 rounded-xl shadow-lg p-8 text-center">
+            <Bookmark className="w-16 h-16 mx-auto text-gray-400 mb-4" />
+            <h2 className="text-2xl font-semibold text-gray-900 dark:text-white mb-2">
+              No saved apprenticeships yet
+            </h2>
+            <p className="text-gray-600 dark:text-gray-400 mb-6">
+              Save apprenticeships you're interested in to view them later.
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.

(react/no-unescaped-entities)